src/start.ts

```diff
@@ -97,11 +103,24 @@ export const handler = async (_event: any = {}): Promise<any> => {
             );
 
             // Fetch keeperSponsorWallet transaction count
-            let keeperSponsorWalletTransactionCount =
-              await provider.getTransactionCount(
-                keeperSponsorWallet.address,
-                currentBlock
+            const retryableGetTransactionCountCall = node.utils.retryOnTimeout(
+              DEFAULT_RETRY_TIMEOUT_MS,
```

Comment:
You use this so often (10x in this file) that it may be worth wrapping `retryOnTimeout`?
Something like:
```
const retryOnTimeout = <T>(operation: () => Promise<T>, options?: ContinuousRetryOptions) =>
  node.utils.retryOnTimeout(DEFAULT_RETRY_TIMEOUT_MS, operation, options);
```
It's strange that the timeout isn't an extra optional argument.

Reply:
Good idea. I'll try using the 2nd option